sphinxcontrib/images.py

```diff
@@ -108,33 +121,24 @@ class ImageDirective(Directive):
     It's backward compatibile and it's adding more cool stuff.
     '''
 
-    align_values = ('left', 'center', 'right')
-
-    def align(argument):
-        # This is not callable as self.align.  We cannot make it a
-        # staticmethod because we're saving an unbound method in
-        # option_spec below.
-        return directives.choice(argument, ImageDirective.align_values)
-
     has_content = True
     required_arguments = True
 
     option_spec = {
         'width': directives.length_or_percentage_or_unitless,
         'height': directives.length_or_unitless,
-        'strech': directives.choice,
```

Comment:
I didn't find any documentation of what this parameter was supposed to do, but it should be `lambda x: directive.choice(x, choices)`

Reply:
lol, I didn't realize it was a misspelling. I just looked for any documentation that used this value, but I may have changed the spelling when searching for it :joy: